File: src/test/java/ru/ezhov/rocket/action/notification/MessageTest.java
package ru.ezhov.rocket.action.notification;

import javax.swing.SwingUtilities;
import java.awt.Dimension;
import java.awt.Point;

public class MessageTest {
    public static void main(String[] args) {
        SwingUtilities.invokeLater(() -> {
            Message message = new Message(4000, "Test");
            message.showMessage(new Point(300, 200), new Dimension(300, 200));
        });
    }
}

File: src/test/kotlin/ru/ezhov/rocket/action/notification/MessageTest.kt
package ru.ezhov.rocket.action.notification

import java.awt.Dimension
import java.awt.Point
import javax.swing.SwingUtilities

object MessageTest {
    @JvmStatic
    fun main(args: Array<String>) {
        SwingUtilities.invokeLater {
            val message = Message(NotificationType.INFO, 4000, "Test")
            message.showMessage(Point(300, 200), Dimension(300, 200))
        }
    }
}